src/screens/WalletScreen/index.js

```diff
@@ -150,6 +220,7 @@ class WalletScreen extends NavigatorComponent<Props & TestingModeProps & Actions
 const mapStateToProps = state => ({
   ...state.wallet,
   testingModeActive: state.testingMode.isActive,
+  networkType: state.accounts.networkType,
```

Comment:
@prameetc There is no field networkType right under `accounts`. As I said you should use [this](https://github.com/Bit-Nation/BITNATION-Pangea-mobile/blob/develop/src/reducers/accounts.js#L151) selector to get current account and then get networkType from it.